src/lib/pickles_types/plonk_types.mli

```diff
@@ -232,12 +232,12 @@ module Openings : sig
          ( 'a
          , 'b
          , 'c
-         , (unit, 'c) Snarky_backendless.Checked_ast.t )
+         , (unit, 'c) Snarky_backendless.Checked_runner.Simple.t )
```

Comment:
We're going to have to do this properly when we finally switch to the rust backend, but I guess this is okay for now.